src/tests/calculator.test.js

```diff
@@ -0,0 +1,33 @@
+import '@testing-library/jest-dom';
+import calculate from '../logic/calculate';
+
+describe('Calculator tests', () => {
+  test('should return an object', () => {
+    // Arrange
+    const object = { total: 10, next: null, operation: '-' };
+    // Act
+    const result = calculate(object, 'AC');
+    // Assert
+    expect(typeof result).toBe('object');
+  });
+
+  test('10-6 should return a total of 4 and the next and operation values should become Null', () => {
+    // Arrange
+    const object = { total: 10, next: 6, operation: '-' };
+    // Act
+    const result = calculate(object, '=');
+    // Assert
+    expect(result.total).toBe('4');
+    expect(result.next).toBeNull();
+    expect(result.operation).toBeNull();
+  });
+
+  test('10 +/- should return -10', () => {
+    // Arrange
+    const object = { total: 10, next: null, operation: null };
+    // Act
+    const result = calculate(object, '+/-');
+    // Assert
+    expect(result.total).toBe('-10');
+  });
+});
```

Comment:
- [ ] @momotsow and @YKerroum great work on your tests however its a project requirement to add tests for all components, kindly add tests for all components
![image](https://user-images.githubusercontent.com/61408860/206546799-3337b155-90aa-4ae0-b94b-fe7509c7b336.png)


- [ ] Kindly ensure all your tests are passing your test for the app component is failing
![image](https://user-images.githubusercontent.com/61408860/206547127-75d7084e-02a0-4582-badf-a49915c5af37.png)

Reply:
- [ ] @team great work implementing some of the changes suggested, however, I can't see tests for the react components, this is a [project requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md), please add the tests
![image](https://user-images.githubusercontent.com/61408860/206557318-bd907d48-926a-4d9f-848e-6dcdd9a5406b.png)